src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1275,7 +1276,7 @@ Namespace Microsoft.CodeAnalysis.Operations
 
         Private Function CreateBoundGotoStatementOperation(boundGotoStatement As BoundGotoStatement) As IBranchOperation
             Dim target As ILabelSymbol = boundGotoStatement.Label
-            Dim branchKind As BranchKind = BranchKind.GoTo
+            Dim branchKind As BranchKind = branchKind.GoTo
```

Comment:
`BranchKind` here and below.

Reply:
Fixed.

---
In reply to: [148031450](https://github.com/dotnet/roslyn/pull/22842#discussion_r148031450) [](ancestors = 148031450)